data/json/bionics.json

```diff
@@ -358,54 +565,108 @@
         "type": "bionic",
         "id": "bio_resonator",
         "name": "Sonic Resonator",
+        "occupied_bodyparts": [
+            [ "TORSO", 15 ],
+            [ "HEAD", 1 ],
+            [ "ARM_L", 2 ],
+            [ "ARM_R", 2 ],
+            [ "LEG_L", 4 ],
+            [ "LEG_R", 4 ]
+        ],
         "act_cost": 100,
         "description": "Your entire body may resonate at very high power, creating a short-range shockwave.  While it will not do much damage to creatures, stiff items such as walls and doors will be damaged."
     },{
         "type": "bionic",
         "id": "bio_time_freeze",
         "name": "Time Dilation",
+        "occupied_bodyparts": [
+            [ "TORSO", 12 ],
+            [ "HEAD", 3 ],
+            [ "ARM_L", 3 ],
+            [ "ARM_R", 3 ],
+            [ "HAND_L", 1 ],
+            [ "HAND_R", 1 ],
+            [ "LEG_L", 4 ],
+            [ "LEG_R", 4 ],
+            [ "FOOT_L", 1 ],
+            [ "FOOT_R", 1 ]
+        ],
         "act_cost": 50,
         "description": "At an immense cost of power, you may increase your body speed and reactions dramatically, essentially freezing time.  You are still delicate, however, and violent or rapid movements may damage you due to friction."
     },{
         "type": "bionic",
         "id": "bio_teleport",
         "name": "Teleportation Unit",
+        "occupied_bodyparts": [
+            [ "TORSO", 16 ],
+            [ "ARM_L", 3 ],
+            [ "ARM_R", 3 ],
+            [ "LEG_L", 4 ],
+            [ "LEG_R", 4 ]
+        ],
         "act_cost": 250,
         "description": "This highly experimental unit folds space over short distances, instantly transporting your body up to 25 feet at the cost of much power.  Note that prolonged or frequent use may have dangerous side effects."
     },{
         "type": "bionic",
         "id": "bio_blood_anal",
         "name": "Blood Analysis",
+        "occupied_bodyparts": [
+            [ "TORSO", 8 ]
+        ],
         "act_cost": 25,
         "description": "Small sensors have been implanted in your heart, allowing you to analyze your blood.  This will detect many illnesses, drugs, and other conditions."
     },{
         "type": "bionic",
         "id": "bio_geiger",
         "name": "Integrated Dosimeter",
+        "occupied_bodyparts": [
+            [ "TORSO", 2 ],
```

Comment:
Thanks for all of these notes, I'll take them into account.
BTW, I'll gather slots data for all bionics a little bit later and provide it in this PR for better balancing of the entire system.